gtsfm/frontend/correspondence_generator/keypoint_aggregator/keypoint_aggregator_dedup.py

```diff
@@ -99,10 +105,11 @@ def aggregate(
             per_image_kpt_coordinates, i2_indices = self.append_unique_keypoints(
                 i=i2, keypoints=keypoints_i2, per_image_kpt_coordinates=per_image_kpt_coordinates
             )
-            putative_corr_idxs = np.stack([i1_indices, i2_indices], axis=-1).astype(np.uint16)
+            putative_corr_idxs = np.stack([i1_indices, i2_indices], axis=-1).astype(np.int32)
```

Comment:
Why this change? indices should be unsigned

Reply:
the gtsam MatchIndicesMap for C++ DSF currently requires int32 input for Eigen (https://github.com/borglab/gtsam/blob/develop/gtsam/sfm/DsfTrackGenerator.h#L32)